core/templates/dev/head/base.html

```diff
@@ -117,11 +117,11 @@
               <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
                 <div class="navbar-container">
                   <div class="navbar-header protractor-test-navbar-header pull-left">
-                    <a ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
+                    <a ng-if="windowWidth < 768" ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
                       <img src="/images/general/menu.png">
                     </a>
                     <a class="oppia-navbar-brand-name oppia-transition-200" href="/gallery" focus-on="<[LABEL_FOR_CLEARING_FOCUS]>">
-                      <img src="/images/logo/288x128_logo_white.png" class="oppia-logo">
+                      <img src="/images/logo/288x128_logo_white.png" ng-class="windowWidth > 768 ? 'oppia-logo-wide' : 'oppia-logo-small'">
```

Comment:
Could you perhaps ng-cloak this (or maybe the contents of the navbar in general)? The reason I ask is that, on my machine, when the page is reloaded, this icon appears really big for a little while, before becoming the correct size after more of the page loads. I think it's because angular takes a while to initialize and we've changed the class to an ng-class.